### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by SWWdz** [sail-sg/mvp#29]
Can I pretrain a model on dataset which is included in our experiment? And I use it to predict keypoints on my custom datasets.

**Comment by twangnh** [comment]
yes, but you will face generalization problem on the custom dataset due to domain shift
